src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorIntegrationTest.java

```diff
@@ -0,0 +1,392 @@
+//                        " -L " + privateTestDir + "ug.random50000.subset300bp.chr1.family.vcf --no_cmdline_in_header -ped " + privateTestDir + "ug.random50000.family.ped -o %s", 1,
+//                Arrays.asList(MD5));
+//        executeTest("Testing ChromosomeCounts annotation with PED file", spec);
+//    }
+//
+    //TODO this has no arguments just yet
+//    @Test(enabled = true)
+//    public void testInbreedingCoeffPed() {
+//        final String MD5 = "95408408863cc81c63ec3c53716bf9d2";
+//        WalkerTestSpec spec = new WalkerTestSpec(
+//                "-T VariantAnnotator -R " + b37KGReference + " -A InbreedingCoeff --variant:vcf " + privateTestDir + "ug.random50000.subset300bp.chr1.family.vcf" +
+//                        " -L " + privateTestDir + "ug.random50000.subset300bp.chr1.family.vcf --no_cmdline_in_header -ped " + privateTestDir + "ug.random50000.family.ped -o %s", 1,
+//                Arrays.asList(MD5));
+//        executeTest("Testing InbreedingCoeff annotation with PED file", spec);
+//    }
+
+    @Test
+    public void testAlleleTrimming() throws IOException {
+        // This test makes sure that the expression code works in a complex case with many overlapping variant contexts
+        assertVariantContextsMatch(getTestFile("AlleleTrim.vcf"),
+                getTestFile("expected/testAlleleTrimming.vcf"),
+                Arrays.asList( "--resource", "exac:" + getToolTestDataDir() + "exacAlleleTrim.vcf", "-E", "exac.AC_Adj", "-A", "InbreedingCoeff"),
+                null, Collections.emptyList());
+    }
+
+    @Test
+    public void testStrandBiasBySample() throws IOException {
+        // Created variants via HalotypeCaller GATK3 with no default annotations
+        final File outputVCF = getTestFile("HCOutputNoAnnotations.vcf");
+
+        // Created variant via HalotypeCaller GATK3; include StrandBiasBySample, exclude FisherStrand annotation
+        //             re-Annotate the variant with VariantAnnotator using FisherStrand annotation
+        final File outputVCFNoFS = getTestFile("HCOutputNoFSAnnotation.vcf");
+
+        final File outputWithAddedFS = createTempFile("variantannotator", ".vcf");
+
+        final ArgumentsBuilder args = new ArgumentsBuilder();
+        args.addReference(new File(b37_reference_20_21));
+        args.addVCF(outputVCFNoFS);
+        args.addOutput(outputWithAddedFS);
+        args.add("-L 20:10130000-10134800");
+        args.add("-A FisherStrand");
+        runCommandLine(args);
+
+        // confirm that the FisherStrand values are identical for the two pipelines
+        final VCFCodec codec = new VCFCodec();
+        final FileInputStream s = new FileInputStream(outputVCF);
+        final LineIterator lineIterator = codec.makeSourceFromStream(new PositionalBufferedStream(s));
+        codec.readHeader(lineIterator);
+
+        final VCFCodec codecAnn = new VCFCodec();
```

Comment:
Use `FeatureDataSource` or `VCFFileReader` here instead of raw codecs and input streams I think.  Also, be sure to put closeables in `try()`